channels-rke2.yaml

```diff
@@ -159,6 +159,10 @@ releases:
         version: 2.11.100-build2021022302
       rke2-multus:
         repo: rancher-rke2-charts
-        version: v3.7.1-build2021041604
-      
-      
+        version: v3.7.1-build2021041604    
+  - version: v1.21.5+rke2r2
+    minChannelServerVersion: v2.6.0-alpha1
```

Comment:
Making the minimum version `v2.6.0-alpha1` is a bigger deal that requires some specific QA sign-off. Here is why. Currently, in the release branch (and this dev branch), v1.21.5+rke2r1 is available for `v2.6.1-alpha1` (see https://github.com/rancher/kontainer-driver-metadata/blob/release-v2.6/channels-rke2.yaml#L120).

With this change, you are now making 1.21.5 available in an older version of rancher, namely 2.6.0. So, we need to make sure that the arguments and charts set forth for v1.21.5 work with 2.6.0. The big one is the addition of the harvester CSI and CPI charts. They were added in 1.21.5. See: 
```
git diff v1.21.4+rke2r2..v1.21.5+rke2r1 -- Dockerfile
diff --git a/Dockerfile b/Dockerfile
index 20f64728..f6541dc7 100644
--- a/Dockerfile
+++ b/Dockerfile
... shortened ...
 RUN CHART_URL="https://github.com/rancher/charts/blob/a1f9fea117dbe1aca21337a1d3b4a3ba2e3d93fd/assets/rancher-vsphere-csi/rancher-vsphere-csi-100.0.0.tgz?raw=true"  CHART_FILE=/charts/rancher-vsphere-csi.yaml CHART_BOOTSTRAP=true /charts/build-chart.sh
+RUN CHART_VERSION="0.1.200"                   CHART_FILE=/charts/harvester-cloud-provider.yaml CHART_BOOTSTRAP=true /charts/build-chart.sh
+RUN CHART_VERSION="0.1.300"                   CHART_FILE=/charts/harvester-csi-driver.yaml     CHART_BOOTSTRAP=true /charts/build-chart.sh
 RUN rm -vf /charts/*.sh /charts/*.md
```

And here's a reference to the release branch showing that the min version for v1.21.5+rke2r1 and the harvester charts is only enabled for 2.6.1: https://github.com/rancher/kontainer-driver-metadata/blob/release-v2.6/channels-rke2.yaml#L119-L142

This change is the impetus for the exchange between @deniseschannon and I that started here: https://suse.slack.com/archives/C02DNASKFQB/p1633476576449500

The takeaway from that is that we CAN release this change OOB to production but ONLY IF we validate that the introduction of the harvester chart doesn't break anything. I have a very strong expectation that it WONT break anything, but it needs verified.